generators/app/gulp-tasks/project.js

```diff
@@ -12,14 +12,12 @@
 
 const mergeStream = require('merge-stream');
 const gulp = require('gulp');
-
 const polymer = require('polymer-build');
-const PolymerProject = polymer.PolymerProject;
-const fork = polymer.forkStream;
-const addServiceWorker = polymer.addServiceWorker;
 
-const polymerJSON = require(global.config.json);
-const project = new PolymerProject(polymerJSON);
+const polymerJSON = require(global.config.polymerJsonPath);
+const project = new polymer.PolymerProject(polymerJSON);
+const bundledPath = `${global.config.build.rootDirectory}/${global.config.build.bundledDirectory}`;
```

Comment:
I think we may want to use `path.join` for these. I *think* (but am not 100% certain) that it will normalize the slashes so we don't end up with filepath errors on Windows